wiki/Reporting_bad_behaviour/Abuse/de.md

```diff
@@ -65,14 +62,20 @@ Wenn der Tweet von einem Konto stammt, welches mit dem Profil eines osu!-Spieler
 
 ## Ein Mitglied eines der Freiwilligenteams (NAT, GMT) nähert sich mir/spricht mit mir/benimmt sich mir gegenüber unangemessen!
 
-Sende **sofort** eine E-Mail mit so viel Informationen wie möglich an [abuse@ppy.sh](mailto:abuse@ppy.sh).
+Sende **sofort** eine E-Mail mit so viel Informationen wie möglich an [abuse@ppy.sh](mailto:abuse@ppy.sh). Dazu gehören Screenshots, Chatverläufe, im Grunde alles, was du an die E-Mail anhängen kannst.
 
-Wenn dir irgendetwas, was du hinzufügen möchtest, peinlich ist, lass' es bitte weg. Dazu gehören Screenshots, Chatverläufe, im Grunde alles, was du an eine E-Mail anhängen kannst.
+Wenn dir irgendetwas, was du hinzufügen möchtest, peinlich ist, lass' es bitte weg.
 
-Wir werden uns damit befassen, sobald wir können.
+Wir werden uns so schnell wie möglich darum kümmern.
 
 ## Jemand, den ich kenne, wird von einem Mitglied eines der Freiwilligenteams (NAT, GMT) missbraucht, hat aber zu viel Angst, sich zu melden!
 
 Sende eine E-Mail an [abuse@ppy.sh](mailto:abuse@ppy.sh), die alles enthält, was du weißt, mit so vielen Beweisen, wie du beschaffen kannst.
 
 Falls du keine Beweise hast, sondern nur Geschichten oder Gerüchte, von denen du glaubst, dass sie stichhaltig genug sind, um darauf zu reagieren, sende uns bitte trotzdem eine E-Mail. Wir nehmen Beschwerden wie diese sehr ernst und werden jeden Bericht eingehend prüfen.
+
+## Meine E-Mail wurde ignoriert!
+
+Wenn du nach einigen Wochen keine Antwort auf deine E-Mail erhalten hast, sende bitte eine weitere E-Mail.
+
+Die E-Mail ging vermutlich unter, aber wir wollen uns trotzdem um deine Meldung kümmern.
```

Comment:
To me, "The email was most likely lost somewhere along the way" implies that the email probably didn't arrive at all (server errors, connection issues etc.) while this sounds more like people just forgot to answer or didn't see it in the inbox. Might be more appropriate to stick closer to the English version here given the topic

Reply:
In my opinion, "along the way" does not focus on the literal meaning here, and can include both technical issues and human errors. While I think my first translation is very common in this context, I can suggest this in response to you

```md
Die E-Mail ging vermutlich unterwegs verloren, ...
```

Reply:
Sounds better IMHO

Reply:
```suggestion
Die E-Mail ging vermutlich unterwegs verloren, aber wir wollen uns trotzdem um deine Meldung kümmern.
```